### ISSUE TITLE
Tiny-ImageNet Reproduction

### ISSUE THREAD

**Issue by NickSpanos55** [khawar-islam/diffuseMix#14]
Hello and thank you for the great work! I am trying to reproduce your results for Tiny-ImageNet, would it possible to share your training hyperparameters (ie, learning rate, optimizer, annealing at which epoch)? I have also noticed that you chose to train for 300 epochs, whereas the original paper (Manifold Mixup) mentions training for 2000, is this because of the number of data generated from the augmentation? Thank you very much!

**Comment by khawar-islam** [comment]
Dear @NickSpanos55 

Thanks for your kind words and sorry for late reply. I used `--lr 0.01`` --optimizer sgd` `--lr_scheduler cosine` `--lr_min` `1e-9` . Yes, we keep 300 epochs and also some papers train models for 3 epochs so we follow same protocols mentioned in previous papers
